### PR TITLE
Modal Component

### PR DESCRIPTION
**Pull request by piercefrawley**
## Changes
- Adds modal component for initial release
- Accepts basic show value, header, body, and footer as props

screen shot 2017-07-12 at 9 37 15 am

### CODE REVIEW

File: layouts/matapihi/src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        </button>
+      </div>
+    );
+  }
+}
+
+render(
+  React.createElement(Component),
+  document.getElementById('root')
+);
```
Comment (chrisarevir) [inline layouts/matapihi/src/index.js]: Interesting, I don't think you need this to explicitly render the modal, but rather just export it?

Comment (chrisarevir) [inline layouts/matapihi/src/index.js]: More like:

```javascript

import Modal from './Modal';

export default Modal;
```

Comment (chrisarevir) [inline layouts/matapihi/src/index.js]: If this logic is needed for the modal to work , I would suggest moving it back to Modal.jsx

Comment (chrisarevir) [inline layouts/matapihi/src/index.js]: Ah, this looks like sandbox copypasta'd over. A mistake perhaps?

Comment (Marswang92) [inline layouts/matapihi/src/index.js]: Oh my bad that's for dev use. I'll fix rn. Thanks!

**Pull request merged**